lms/static/sass/shared/_footer-edx.scss

```diff
@@ -628,6 +981,81 @@ footer#footer-edx-v3 {
       @include border-left(1px solid $courseware-button-border-color);
     }
 
+    .column-1 {
+      flex: 0 0 16.6667%;
+      max-width: 16.6667%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-2 {
+      flex: 0 0 50%;
+      max-width: 50%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-3 {
+      flex: 0 0 33.3333%;
+      max-width: 33.3333%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-2-1 {
+      flex: 0 0 33.3333%;
+      max-width: 33.3333%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+      //flex-basis: auto;
+    }
+
+    .column-2-2 {
+      flex: 0 0 80%;
+      max-width: 80%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+
+      @include margin-left(0%);
+      //flex-basis: auto;
+    }
+
+    .column-3-1 {
+      flex: 0 0 100%;
+      max-width: 100%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-3-2 {
+      flex: 0 0 100%;
+      max-width: 100%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
```

Comment:
Please update to be DRY.

Reply:
Most of these styles are the defaults for the column and haven't changed so there is no need to declare again.